graphql-kotlin-types/src/main/kotlin/com/expediagroup/graphql/operations/Query.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.expediagroup.graphql.spring.operations
+package com.expediagroup.graphql.operations
```

Comment:
Should we create a separate `com.expediagroup.graphql.server` package as this marker interfaces are only used for the server?

Reply:
👍 sure, I'll move this change into separate PR.